packages/analytics-middleware/transformers/publish/story.js

```diff
@@ -18,7 +18,7 @@ const removeNullObjectKeys = (data) => {
 
   if (data) {
     Object.keys(data)
-      .filter(key => data[key] !== null)
+      .filter(key => data[key] !== null && typeof data[key] !== 'undefined')
```

Comment:
```suggestion
      .filter(key => data[key] != null)
```
Would this be the same here?